pyV2DL3/tests/test_fillRESPONSE.py

```diff
@@ -0,0 +1,24 @@
+import logging
+import pytest
+
+from pyV2DL3.eventdisplay.fillRESPONSE import check_fuzzy_boundary
+
+
+def test_check_fuzzy_boundary(caplog):
```

Comment:
Very nice! What about a test when par == boundary?

Reply:
`check_funzzy_boundary` function is not called  when `par == boundary` case occurs since this function is inside ` if np.all(irf_stored_par < par) or np.all(irf_stored_par > par):`.

Do you still suggest to have this case in test function?